routes/index.js

```diff
@@ -1,9 +1,27 @@
 var express = require('express');
 var router = express.Router();
+var mysql = require('mysql');
+var app = express()
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+  var connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'root',
+    password: 'Kimleng123$',
+    database: 'cs157a',
+  });
+  
+  connection.connect()
+  
+  connection.query('SELECT * FROM emp', function(err, rows, fields) {
+    if (err) throw err
+    res.render('index', {title: "id: " + rows[0]["id"] + ", " 
+    + "name: " + rows[0]["name"] + ", " 
+    + "age:" + rows[0]["age"]})
+  })
```

Comment:
This works, which is good. Can we improve it to list all the rows and not just row 0? Chances are, the TA will tell us to add something to the DB and see if it shows up on the webpage.